frontend/src/v4/routes/subscription/components/subscriptionForm.component.tsx

```diff
@@ -355,7 +355,7 @@ export class SubscriptionForm extends PureComponent<IProps, IState> {
 								<FormInfo>** Subject to VAT where applicable</FormInfo>
 							</FormInfoContainer>
 							<PayPalInfoContainer>
-								<ConfirmContainer>
+								{/* <ConfirmContainer>
```

Comment:
if you were to comment it out you might as well remove it.

I don't think it ever enables anyway though?

Reply:
ok, removed.